### DISCUSSION TITLE
How to configure conda env in pyrightconfig.json file ?

### DISCUSSION THREAD

**Discussion by rkscodes** [zed-industries/zed#6561] [Feature Requests]
I am using mac app and can't really activate my conda env for the projects is there some way ?

**Comment by danielcs88** [comment]
> I am using mac app and can't really activate my conda env for the projects is there some way ?

@rkscodes

I was able to use Conda by creating a file in the directory I'm working by creating a file called `pyrightconfig.json`

```json
{
  "executionEnvironments": [
    {
      "python": "<path_to_conda_env>/bin/python"
    }
  ]
}
```

What I don't know yet is how to configure for all Zed instances rather than having to create a `pyrightconfig.json` in _every_ project directory.

**Reply by rkscodes** [reply]
I tried it again, but zed would not resolve my packages in editor seems like env is not active after setting this config

**Comment by dwinkler1** [comment]
Does this work for you:

https://github.com/zed-industries/zed/issues/7646#issuecomment-1975170247

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
